src/commands/schema/ls.js

```diff
@@ -0,0 +1,41 @@
+const FaunaCommand = require("../../lib/fauna-command.js");
+const fetch = require("node-fetch");
+
+class ListSchemaCommand extends FaunaCommand {
```

Comment:
minor: I'm assuming this lists all files, even one in nested directories? if so, we might want to name it something other than `ls`, as that isn't how `ls` behaves in the shell.

Reply:
It lists the schema files for one database and not for databases nested inside, so I think it behaves like `ls` in that way.